src-built-in/components/assets/css/_fontIcons.css

```diff
@@ -1,2 +1,24 @@
-@import './finfont.css';
-@import './_fontIconWhitelabel.css';
+@import './font-finance.css';
+@import './_fontIconWhiteLabel.css';
```

Comment:
@DanNicolai if you put these classes after _fontIconWhiteLabel.css, then there is no way to actually override them.